src/app/econ/supply-demand/lesson/SupplyDemandLesson.tsx

```diff
@@ -0,0 +1,748 @@
+              text: "The quantity will decrease",
+              isCorrect: false,
+              explanation: "",
+            },
+            {
+              text: "The demand will increase",
+              isCorrect: true,
+              explanation: "",
+            },
+            {
+              text: "The demand will decrease",
+              isCorrect: false,
+              explanation: "",
+            },
+          ]}
+        ></MultiSelectQuizQuestion>
+      </Block>
+
+      <Block color="purple" title="Question 3">
+        <TextQuizQuestion
+          question={
+            <>
+              If there is a groundbreaking new technology that makes french
+              fries cheaper to produce, <br />
+              <br /> the price of french fries will:{" "}
+            </>
+          }
+          placeholder="e.g. increase/decrease/stay the same"
+          pattern={`decrease`}
+        />
+        <TextQuizQuestion
+          question="The quantity of french fries will:"
+          placeholder="e.g. increase/decrease/stay the same"
+          pattern={`increase`}
+        />
+      </Block>
+      <Block color="green" title="Question 4">
+        <QuizQuestion
+          question="If the price of french fries goes up, what happens on the supply and demand graph?"
+          choices={[
+            {
+              text: "The entire supply or demand curve will shift.",
+              isCorrect: false,
+              explanation:
+                "No, only outside factors (not price) shift the entire curve.",
+            },
+            {
+              text: "Our point on the supply and demand graph will move along the curve.",
+              isCorrect: true,
+              explanation:
+                "Correct! A change in price moves the point along the curve, not the curve itself.",
+            },
+            {
+              text: "The graph will disappear.",
+              isCorrect: false,
+              explanation: "No, the graph remains; only the point moves.",
+            },
+          ]}
+        />
+      </Block>
+    </Block>,
+    <Block color="yellow" title="Conclusion">
+      <p>
+        In this lesson, we learned about the supply and demand model and how to
+        use it to set the best price to make the most money.
+      </p>
+      <ColorBox color="green">
+        <Emphasize>
+          Here are some questions to ponder about as you continue your Econ
+          journey:
+        </Emphasize>
+        <br />
+        - Can you create a situation where price stays the same but quantity
+        increases?
+        <br />
+        - Is it possible to shift both curves and keep the equilibrium
+        unchanged?
+        <br />
+        - Can you create a situation where price goes up and quantity goes down?
+        <br />
+      </ColorBox>
+      <p>
+        Thinking about these questions will help you understand not only the
+        supply and demand model, but also the real world.
```

Comment:
grammar nit: I think the second clause of this sentence needs to be an independent clause. Since it's the last one, I'd rephrase